### ISSUE TITLE
xy-VSFilter builds based on xy_sub_filter_rc* might be buggy

### ISSUE THREAD

**Issue by TheOneric** [pinterf/xy-VSFilter#38]
Just to let you know, haven't checked if you already identified and fixed any such rendering issues:

In https://github.com/Cyberbeing/xy-VSFilter/pull/18 Cyberbeing mentioned there are differences between upstream’s `xy_sub_filter_rc*` XySubFilter branches and the ones intended for xy-VSFilter and xy-VSFilter binaries based on the XySubFilter branches might be buggy. See https://github.com/Cyberbeing/xy-VSFilter/pull/18#issuecomment-1302638201 and follow-ups.

On a cursory glance it appears like you are using the XySubFilter branch for both XySubFilter and xy-VSFilter builds. At least your builds are not containing upstream’s CSRI name change from 2014 which was only committed to the xy-VSFilter branch: https://github.com/Cyberbeing/xy-VSFilter/commit/0d64ad7bc03ab26d68150670ef92f3105b24ee20 (released in [3.0.0.306](https://github.com/Cyberbeing/xy-VSFilter/releases/tag/3.0.0.306))

**Comment by NBruderman** [comment]
@pinterf can you push that commit, just so the branches sync up?

**Comment by TheOneric** [comment]
> push that commit, just so the branches sync up?

There are supposedly more differences between Cyberbeing’s `master` (xy-VSFilter) and `xy_sub_filter_rc*` (XySubFilter) branches. Whether pinterf already merged all other fixes I do no know. If not, pulling in just the name change is a start but not a full solution.

**Comment by NBruderman** [comment]
Yeah, I agree... I guess the solution is either waiting for pinterf to check and merge the missing patches, or going yourself and checking the differences, and finding what changes weren't merged...
